go/libkb/interfaces.go

```diff
@@ -135,7 +135,7 @@ type CommandLine interface {
 type Server interface {
 }
 
-type DBKeySet map[DbKey]bool
+type DBKeySet map[DbKey]struct{}
```

Comment:
what is this change for?

Reply:
It shouldn't break nor fix anything. I think it's a slight improvement. With `struct` it is unambiguous how the `map` represents a set and all values are valid. With `bool` a `false` value is possible, and it's ambiguous whether that means it's in the set. With `struct` no one will be tempted to write `someset[key] = false`